syncr_frontend/frontend.py

```diff
@@ -19,6 +20,20 @@
 # Backend Access Functions
 
 
+def send_message(message):
+    # Sends given message to backend
```

Comment:
Use proper docstring format (start with `"""` on the first line, end with `"""` on the last line).

Reply:
Tryjankowski's on the road right now so I'll convert these comments to docstring format for him. I'd like to squash and merge his stuff right now so I can pull and do the necessary merge conflict fixin's